microSALT/cli.py

```diff
@@ -43,7 +43,8 @@
     sys.exit(-1)
 
 
-def set_cli_config(config):
+@click.pass_context
+def set_cli_config(ctx, config):
```

Comment:
Was this function broken previously? Where did it get ctx from?

Reply:
It apparently didn't get it from anywhere. Flake8 complained that it was undefined, and I haven't been able to find anything else (fixture-like) that would explain the definition. Though, I might be missing if something "magical" is going on here!